documentation/DeveloperGuide.md

```diff
@@ -32,15 +32,21 @@ actually use Clion for editing, VSCode for debugging and Nuclide for doing sourc
 
 ## Running
 
-Running `cargo build` will produce a binary at `target/debug/mirai`. Create a handy alias for the binary with
+Running `cargo build` will produce a binary at `target/debug/mirai`.
+ 
+Unfortunately cargo build sets the dynamic library load path (rpath) that is linked into the binary to a path that is
+invalid a when the binary is run. If you run the binary via cargo `cargo run -- <args>` then cargo overrides the bad
```

Comment:
"a when" -> "when"

   If you run the binary via cargo `cargo run -- <args>` then cargo overrides the bad

->

   If you run the binary via `cargo run -- <args>` then cargo overrides the bad